File: TsogosunProfileAdminWeb/src/app/model/Dtos/ms-patron-details/gmsConfirmedPatronDetailsDto.ts
import { ConfirmedPatronDetails } from "./confirmed-patron/confirmedPatronDetails";

export interface GMSConfirmedPatronDetailsDto extends ConfirmedPatronDetails {
    countryID: number;
    provID: number;
    pcId: number;
    documentTypeId: number;
    documentExpireDate?: Date | undefined;  
    pippep: number;
}

File: TsogosunProfileAdmin/MSPatronDetails/Model/Dtos/GMSConfirmedPatronDetailsDto.cs

using System;
using tsogosun.com.MSPatronDetails.Model.Dtos.ConfirmedPatron;

namespace tsogosun.com.MSPatronDetails.Model.Dtos
{
    public class GMSConfirmedPatronDetailsDto : ConfirmedPatronDetails
    {       
        public int CountryID { get; set; }
        public int ProvID { get; set; }
        public int PcId { get; set; }
        public int DocumentTypeId { get; set; }
        public DateTime? DocumentExpireDate { get; set; }
        public int? Pippep { get; set; }
        
    }
}
